### PR TITLE
20161127 ml

### PR DESCRIPTION
**Pull request by LeeRedden**

### CODE REVIEW

**Review by LeeRedden — COMMENTED** [review]
not that great, nice functionality

**Comment by wgrossman** [comment]
My first comment. What fun.

File: code/reader.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        img = cv2.imread(path)
+        gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+        resize = cv2.resize(gray, (self.width, self.height), interpolation=cv2.INTER_CUBIC)
+        reshape = resize.reshape((1, self.height*self.width))
+        return reshape
+
+
+    def next_batch(self, batch_size):
+        self.count += batch_size
+
```
Comment (wgrossman) [inline code/reader.py]: This is just terrible. TODO: Fix this shit.

File: code/reader.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        	class_name = self.classNames[class_idx]
+        	for file_name in self.classes[class_name]:
+        		path = os.path.join(self.folder, class_name, file_name)
+        		labels[idx,class_idx] = 1
+        		data[idx,:] = self._read_image(path)
+        		idx += 1
+
+        return data, labels
+
+
```
Comment (wgrossman) [inline code/reader.py:100]: WTF?

**Review by wgrossman — APPROVED** [review]
Exceeds expectations.

**Pull request merged**